<!DOCTYPE html> <html> <head> <meta charset=utf-8> <meta name=author content="Fritz Lekschas"> <meta name=description content="Visual exploration of large genome interaction matrices with interactive small multiples."> <meta name=keywords content="Matrix Visualization, Hi-C, Genome Interaction Matrix, Interactive Small Multiples, Genome Contact Maps, Biomedical Visualization, Information Visualization, Chromosome Conformation Capture, Harvard University, Harvard John A. Paulson School of Enbgineering and Applied Sciences, Harvard Medical School, Visual Computing Group, Gehlenborg Lab"> <meta name=viewport content="width=device-width,initial-scale=1"> <title>HiPiler</title> <link rel=apple-touch-icon href=images/touchicon.png> <link rel=icon href=images/favicon.png> <!--[if IE]><link rel="shortcut icon" href="images/favicon.ico"><![endif]--> <meta name=msapplication-TileColor content=#FFFFFF> <meta name=msapplication-TileImage content=images/tileicon.png> <link href="//fonts.googleapis.com/css?family=Rubik:300,400,500" rel=stylesheet type=text/css> <script async src="https://www.googletagmanager.com/gtag/js?id=UA-1367902-4"></script> <script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","UA-1367902-4")</script> <link href="/index.css" rel="stylesheet"></head> <body> <div class=content-wrapper> <div class="content p-t-2 p-b-2"> <div id=bg-img><div class=fade-out></div></div> <h1 class=on-top> <svg id=logo viewBox="0 0 16 16" class=logo> <path id=logo-1 d="M8 8.25L.75 11.633 8 15.017l7.25-3.384L8 8.25z" fill=#fff stroke=currentColor stroke-width=.5 opacity=0.7 /> <path id=logo-2 d="M8 6.8L.75 10.183 8 13.567l7.25-3.384L8 6.8z" fill=#fff stroke=currentColor stroke-width=.5 opacity=0.75 /> <path id=logo-3 d="M8 5.35L.75 8.733 8 12.117l7.25-3.384L8 5.35z" fill=#fff stroke=currentColor stroke-width=.5 opacity=0.8 /> <path id=logo-4 d="M8 3.9L.75 7.283 8 10.667l7.25-3.384L8 3.9z" fill=#fff stroke=currentColor stroke-width=.5 opacity=0.85 /> <path id=logo-5 d="M8 2.45L.75 5.833 8 9.217l7.25-3.384L8 2.45z" fill=#fff stroke=currentColor stroke-width=.5 opacity=0.9 /> <path id=logo-6 d="M8 1L.75 4.383 8 7.767l7.25-3.384L8 1z" fill=#fff stroke=currentColor stroke-width=.5 opacity=0.95 /> <path id=logo-6-b d="M8 1L.75 4.383 8 7.767l7.25-3.384L8 1z" fill=none stroke=currentColor stroke-width=.5 /> <path id=logo-6-x d="M11.625 6.075l-7.25-3.383M4.375 6.075l7.25-3.383" fill=none stroke=currentColor stroke-width=.5 stroke-linecap=butt /> </svg> HiPiler </h1> <h2 class=on-top>Visual exploration<br/>of large genome interaction matrices<br/>with interactive small multiples.</h2> <video id=fig-teaser loop poster=images/video-first-frame.jpg class="bg-video on-top"> <source src=videos/about.webm type=video/webm> <source src=videos/about.mp4 type=video/mp4> </video> <a id=demo-link href=http://hipiler.higlass.io target=_blank>Live Demo</a> <p class="abstract on-top"> HiPiler is an interactive web application for exploring and visualizing many features in large <a href=https://en.wikipedia.org/wiki/Chromosome_conformation_capture#Hi-C_.28all-vs-all.29 target=_blank>genome interaction matrices</a>. Genome interaction matrices approximate the physical distance of pairs of genomic regions to each other and can contain up to 3 million rows and columns. Traditional matrix aggregation or pan-and-zoom interfaces largely fail in supporting search, inspection, and comparison of local features. HiPiler represents features as thumbnail-like <em>snippets</em>. Snippets can be laid out automatically based on their data and meta attributes. They are linked back to the matrix, visualized with <a href=http://higlass.io target=_blank>HiGlass</a>, and can be explored interactively. </p> <h3 id=news class="iconized underlined anchored on-top"> <a href=#news class=hidden-anchor> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#link></use> </svg> </a> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#new></use> </svg> <span>News</span> </h3> <ol class="no-list-style large-spacing on-top"> <li> <a href=http://gehlenborg.com target=_blank>Nils</a> presented HiPiler among other tools at <a href=https://vizbi.org/2018 target=_blank>Visualizing Biologival Data (VIZBI) 2018</a>. <span class=data-links>[<a href=https://www.slideshare.net/ngehlenborg/visualization-of-3d-genome-data target=_blank>Slides</a>]</span> <em class=date>Mar 28th, 2018</em> </li> <li> Version 1.2 is <a href=https://github.com/flekschas/hipiler/releases/tag/v1.2.0 target=_blank>out</a>, supporting five color maps with custom color scaling and featuring HiGlass version 0.10.11 <a href=https://github.com/flekschas/hipiler/blob/master/CHANGELOG.md target=_blank>among other things</a>. <em class=date>Jan 21st, 2018</em> </li> <li> <a href=http://gehlenborg.com target=_blank>Nils</a> talked about HiPiler at <a href=https://www.hsph.harvard.edu/2017-pqg-conference/ target=_blank>Harvard's 2017 PQG Conference</a>. <span class=data-links>[<a href=https://www.slideshare.net/ngehlenborg/higlass-hipiler-making-sense-of-chromosome-interaction-data-with-multiscale-data-visualization-tools-81568229 target=_blank>Slides</a>]</span> <em class=date>Nov 2nd, 2017</em> </li> <li> <a href=https://lekschas.de target=_blank>Fritz</a> presented HiPiler at <a href=https://www.microsoft.com/en-us/research/event/cabi-2017/ target=_blank>Microsoft's Computational Aspects of Biological Information 2017</a>. <span class=data-links>[<a href=https://vcg.seas.harvard.edu/publications/hipiler-visual-exploration-of-large-genome-interaction-matrices-with-interactive-small-multiples/higlass-hipiler-cabi-2017.pdf target=_blank>Poster</a>]</span> <em class=date>Oct 13th, 2017</em> </li> <li> <a href=https://lekschas.de target=_blank>Fritz</a> presented HiPiler at <a href=http://ieeevis.org/year/2017/info/overview-amp-topics/papers-sessions target=_blank>IEEE VIS</a> in Phoenix at InfoVis. <span class=data-links>[<a href=https://vimeo.com/238502606 target=_blank>Presentation</a> | <a href=https://speakerdeck.com/flekschas/hipiler-visual-exploration-of-large-genome-interaction-matrices-with-interactive-small-multiples target=_blank>Slides</a>]</span> <em class=date>Oct 5th, 2017</em> </li> <li> <a href=https://lekschas.de target=_blank>Fritz</a> was invited to pitch HiPiler at Harvard - Novartis Machine Learning and Computational Meeting. <em class=date>Sep 22nd, 2017</em> </li> <li> Nature published an <a href=http://blogs.nature.com/naturejobs/2017/09/11/techblog-hipiler-simplifies-chromatin-structure-analysis/ target=_blank>entire blog post</a> about HiPiler. <em class=date>Sep 11th, 2017</em> </li> <li> <a href=https://lekschas.de target=_blank>Fritz</a> demonstrated HiPiler at <a href=https://www.4dnucleome.org target=_blank>4D Nucleome</a> Annual Meeting 2017. <span class=data-links>[<a href=https://speakerdeck.com/flekschas/hipiler-exploring-many-hi-c-features-through-visual-decomposition target=_blank>Slides</a> | <a href=https://vcg.seas.harvard.edu/publications/hipiler-visual-exploration-of-large-genome-interaction-matrices-with-interactive-small-multiples/poster target=_blank>Poster</a>]</span> <em class=date>Sep 7th, 2017</em> </li> <li> HiPiler got accepted at <a href=http://ieeevis.org/year/2017/info/overview-amp-topics/papers-sessions target=_blank>IEEE VIS (InfoVis)</a>. <em class=date>Jul 11th, 2017</em> </li> </ol> <h3 id=screencast class="iconized underlined anchored"> <a href=#screencast class=hidden-anchor> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#link></use> </svg> </a> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#video></use> </svg> <span>Screencast</span> </h3> <div id=youtube class=video> <div id=youtube-play class="placeholder-btn flex-c flex-jc-c flex-a-c"> <svg class=icon viewBox="0 0 16 16"> <path d="M15.84 4.756s-.156-1.102-.636-1.588c-.608-.638-1.29-.64-1.604-.678-2.24-.162-5.6-.166-5.6-.166s-4.16.036-5.44.16c-.356.067-1.156.047-1.764.684C.316 3.655.16 4.755.16 4.755S0 6.05 0 7.345v1.26c0 1.295.16 2.59.16 2.59s.156 1.1.636 1.587c.608.637 1.408.616 1.764.684 1.28.123 5.44.16 5.44.16s3.36-.004 5.6-.166c.314-.038.996-.04 1.604-.678.48-.486.636-1.59.636-1.59S16 9.9 16 8.605V7.346c0-1.296-.16-2.59-.16-2.59z" class=secondary /> <path d="M6.517 4.8l4.176 2.588c.94.58.924.58-.013 1.156L6.527 11.13c-.9.585-.898.398-.898-.352V5.155c-.002-.688-.002-.937.887-.355z" class=primary /> </svg> </div> </div> <h3 id=introduction class="iconized underlined anchored"> <a href=#introduction class=hidden-anchor> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#link></use> </svg> </a> <svg class="icon icon-small"> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#text></use> </svg> <span>Introduction</span> </h3> <section class=text> <div class=columns> <p class=column-1-2> The human genome is about 2 meters long and tightly folded into each cell nucleus. This results in a dense, fractal-like and three-dimensional structure in which genome sequences that are distant on the genome, can be in close spatial proximity. It has been shown [<a href=#ref-1>1</a>] that this 3D structure is an important factor for regulation of gene expression, replication, DNA repair, and other biological functions. Biologists are interested in uncovering the mechanisms that drive global and local folding to better understand the vast and complex gene regulation network. </p> <p class=column-1-2> The probability of two sequences being in close proximity to each other, i.e. interacting, can be inferred using modern genome sequencing techniques, which yield for every genome a huge symmetric genome interaction matrix with up to 3 million rows and 3 million columns (<a href=#fig-hic>Fig. 1</a>). Each of the 9 trillion matrix cells represents the proximity of two genomic regions. Repetitive and hierarchically nested visual patterns (<a href=#fig-patterns>Fig. 2</a>), ranging from millions down to a few thousand base pairs in size, can be identified across the matrix, which represent so called regions of interest (ROIs). </p> </div> <div class=columns> <div class=column-1-2> <figure id=fig-hic> <img class=zoomable src=/images/fig-hic.jpg data-zoom-target=images/fig-hic-hd.jpg alt="Fig. 1: Hi-C"> <figcaption>Fig. 1: Hi-C methodology: as the DNA (1) is organized non-arbitrarily in the cell nucleus (2), certain parts (highlighted in orange and blue) are frequently in close contact (3). These contacts are quantified over a set of several hundred million cells (4), leading to interaction matrix of up to 3 by 3 million cells (5). Dark colors indicate more frequent contact occurrences of two loci.</figcaption> </figure> </div> <div class=column-1-2> <figure id=fig-patterns> <img class=zoomable src=/images/fig-patterns.jpg data-zoom-target=images/fig-patterns-hd.jpg alt="Fig. 2: Hi-C Patterns"> <figcaption>Fig. 2: Examples of frequent patterns by increasing size. Loops (1) appear as dark central dots. Hierarchically-organized domains (2) are darker rectangles. Flames (3) are horizontal or vertical lines. Active and inactive compartments create a global checkerboard pattern (4).</figcaption> </figure> </div> </div> <div class="columns m-t-2 m-b-2"> <p class=column-1-2> Algorithms for automatic pattern detection are being development. However, these algorithms can be very complex and often identify tens of thousands of pattern instances. Results of algorithms designed to identify the same type of pattern often differ substantially [<a href=#ref-2>2</a>] and the lack of a ground-truth pattern collection hinders thorough evaluation of these algorithms. </p> <p class=column-1-2> Interactive visualization tools have been developed [<a href=#ref-3>3</a>] but are focused on supporting visualization of a single or a small number of views of the matrix and navigation through pan and zoom [<a href=#ref-4>4</a>, <a href=#ref-5>5</a>]. Detailed exploration and comparison of thousands of small ROIs is unsupported by current tools yet needed, due to the size and multiscale nature of the folded genome. </p> </div> <p class="abstract m-t-2 m-b-2"> HiPiler is an interactive visualization tool designed for exploration and analysis of thousands of ROIs extracted from one or more genome interaction matrices. </p> <div id=Technique class="columns m-t-0-5 m-b-1"> <p class=column-1-2> To overcome the contextual constraints of exploring local patterns in large matrices, HiPiler follows a divide and explore approach that extracts ROIs from the matrix and enables independent exploration (<a href=#fig-approach>Fig. 3</a>). HiPiler assumes a given set of ROIs, derived from specialized pattern recognition algorithms. HiPiler then visualizes these ROIs as small heatmaps (matrices) which we call <em>snippets</em>. A snippet is associated with a set of ordinal and categorical attributes, such as its noisiness, size, or source dataset. This data is derived from the matrix itself or point to prior knowledge. Based on this data, HiPiler enables automatic and manual ordering, positioning, grouping, filtering, and visual manipulation to identify patterns present across the set of snippets. Additionally, the context of snippets in the matrix is maintained through highlighting of snippet locations in the interaction matrix. </p> <div class=column-1-2> <figure id=fig-approach class=m-t-0> <img class=zoomable src=/images/fig-approach.jpg data-zoom-target=images/fig-approach-hd.jpg alt="Fig. 3: The snippets approach."> <figcaption>Fig. 3: The <em>snippets</em> approach: decompose a large matrix (1) into small snippets (2) and explore these snippets (3) using different layouts, arrangements, and styles, while maintaining global context.</figcaption> </figure> <p> The design of HiPiler (<a href=#fig-design>Fig. 4</a>) is informed by semi-structured interviews with ten domain experts from various genomics research labs as well as iterative design sessions over the course of several months. These interviews led to the formulation of six tasks for the exploration of many ROIs in large matrices. </p> </div> </div> <figure id=fig-design> <img class=zoomable src=/images/fig-design.jpg data-zoom-target=images/fig-design-hd.jpg alt="Fig. 4: Design of visualization and interaction concepts in HiPiler."> <figcaption>Fig. 4: Design of visualization and interaction concepts in HiPiler.</figcaption> </figure> <div id=evaluation class="columns m-t-0-5 m-b-1"> <p class=column-1-2> HiPiler is designed to support four types of scenarios. (i) visual evaluation of the results of pattern detection algorithms. (ii) characterization, aggregation, and outlier detection in large pattern collections (<a href=#fig-filter-compare-correlate>Fig. 5.1</a>). (iii) comparison of ROIs across multiple matrices (<a href=#fig-filter-compare-correlate>Fig. 5.2</a>), e.g., to compare different datasets, experimental conditions, or extraction algorithms. </p> <p class=column-1-2> And (iv) correlation of matrix patterns with other genomic attributes (<a href=#fig-filter-compare-correlate>Fig. 5.3</a>), e.g., genes or protein-binding sites. We evaluated the usability and appropriateness of HiPiler through a user study with five domain experts. The results show that HiPiler is easy to learn and use, and that it offers important benefits for analyzing genome interaction matrices. </p> </div> <figure id=fig-filter-compare-correlate> <img class=zoomable src=/images/fig-filter-compare-correlate.jpg data-zoom-target=images/fig-filter-compare-correlate-hd.jpg alt="Fig. 5: Design of HiPiler"> <figcaption>Fig. 5: Illustrations on how HiPiler supports (1) filtering and grouping, (2) comparison, and (3) correlation.</figcaption> </figure> </section> <h3 id=publications class="iconized underlined anchored"> <a href=#publications class=hidden-anchor> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#link></use> </svg> </a> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#document></use> </svg> Publication </h3> <ol id=publications class=no-list-style> <li class=reference> <p class=title><a href=https://doi.org/10.1109/TVCG.2017.2745978 target=_blank>HiPiler: Visual Exploration Of Large Genome Interaction Matrices With Interactive Small Multiples</a></p> <div class=secondary-information> <ol class="no-list-style inline-list authors"> <li>Fritz Lekschas</li> <li>Benjamin Bach</li> <li>Peter Kerpedjiev</li> <li>Nils Gehlenborg</li> <li>Hanspeter Pfister</li> </ol> <span class=journal>IEEE Transactions on Visualization and Computer Graphics</span> (InfoVis), 24, 1, 522-531, 2018. </div> </li> </ol> <h3 id=presentations class="iconized underlined anchored"> <a href=#presentations class=hidden-anchor> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#link></use> </svg> </a> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#presentation></use> </svg> <span>Presentations</span> </h3> <div class=flex-c> <div class="flex-g-1 column-1-2"> <strong>For VIS researchers:</strong> </div> <div class="flex-g-1 column-1-2 m-l-1"> <strong>For biomedical researchers:</strong> </div> </div> <div class=flex-c> <div id=presentation-vis class="flex-g-1 presentation-container"></div> <div id=presentation-bio class="flex-g-1 presentation-container"></div> </div> <div class="flex-c smaller"> <p class="flex-g-1 column-1-2"> IEEE VIS InfoVis, Phoenix, 2017 </p> <p class="flex-g-1 column-1-2 p-l"> 4D Nucleome Annual Meeting, Bethesda, 2017 </p> </div> <h3 id=source-code class="iconized underlined anchored"> <a href=#source-code class=hidden-anchor> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#link></use> </svg> </a> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#code></use> </svg> Source Code </h3> <p>All the code of HiPiler are publicly accessible and open-source.</p> <ul class="no-list-style large-spacing iconized smaller"> <li class=iconized> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#github></use> </svg> <a href=https://github.com/flekschas/hipiler target=_blank>https://github.com/flekschas/hipiler</a> </li> <li class=iconized> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#github></use> </svg> <a href=https://github.com/hms-dbmi/higlass-server target=_blank>https://github.com/hms-dbmi/higlass-server</a> </li> </ul> <h3 id=authors class="iconized underlined anchored"> <a href=#authors class=hidden-anchor> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#link></use> </svg> </a> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#people></use> </svg> Authors </h3> <ol id=authors-list class=no-list-style> <li> <h4>Fritz Lekschas</h4> <p>Harvard John A. Paulson School of Engineering and Applied Sciences</p> <ul class="no-list-style inline-list iconized remix-links"> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#globe></use> </svg> <a href=https://lekschas.de target=_blank>https://lekschas.de</a> </li> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#twitter></use> </svg> <a href=https://twitter.com/flekschas target=_blank>flekschas</a> </li> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#github></use> </svg> <a href=https://github.com/flekschas target=_blank>flekschas</a> </li> </ul> </li> <li> <h4>Benjamin Bach</h4> <p>University of Edinburgh</p> <ul class="no-list-style inline-list iconized remix-links"> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#globe></use> </svg> <a href=http://benjbach.me target=_blank>http://benjbach.me</a> </li> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#twitter></use> </svg> <a href=https://twitter.com/benjbach target=_blank>benjbach</a> </li> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#github></use> </svg> <a href=https://github.com/benjbach target=_blank>benjbach</a> </li> </ul> </li> <li> <h4>Peter Kerpedjiev</h4> <p>Harvard Medical School</p> <ul class="no-list-style inline-list iconized remix-links"> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#globe></use> </svg> <a href=http://emptypipes.org/ target=_blank>http://emptypipes.org</a> </li> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#twitter></use> </svg> <a href=https://twitter.com/pkerpedjiev target=_blank>pkerpedjiev</a> </li> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#github></use> </svg> <a href=https://github.com/pkerpedjiev target=_blank>pkerpedjiev</a> </li> </ul> </li> <li> <h4>Nils Gehlenborg</h4> <p>Harvard Medical School</p> <ul class="no-list-style inline-list iconized remix-links"> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#globe></use> </svg> <a href=http://gehlenborglab.org target=_blank>http://gehlenborglab.org</a> </li> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#twitter></use> </svg> <a href=https://twitter.com/nils_gehlenborg target=_blank>nils_gehlenborg</a> </li> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#github></use> </svg> <a href=https://github.com/ngehlenborg target=_blank>ngehlenborg</a> </li> </ul> </li> <li> <h4>Hanspeter Pfister</h4> <p>Harvard John A. Paulson School of Engineering and Applied Sciences</p> <ul class="no-list-style inline-list iconized remix-links"> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#globe></use> </svg> <a href=https://vcg.seas.harvard.edu target=_blank>https://vcg.seas.harvard.edu</a> </li> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#twitter></use> </svg> <a href=https://twitter.com/hpfister target=_blank>hpfister</a> </li> <li> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#github></use> </svg> <a href=https://github.com/hpfister target=_blank>hpfister</a> </li> </ul> </li> </ol> <h3 id=references class="iconized underlined anchored"> <a href=#references class=hidden-anchor> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#link></use> </svg> </a> <svg class=icon> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#bookmark></use> </svg> References </h3> <ol class="no-list-style large-spacing iconized"> <li id=ref-1 class="flex-c reference"> <span class=number>[1]</span> <span class=flex-g-1>Fraser and Bickmore. (2017) Nuclear organization of the genome and the potential for gene regulation. <em>Nature</em>, 447, 7143, 413–417.</span> </li> <li id=ref-2 class="flex-c reference"> <span class=number>[2]</span> <span class=flex-g-1>Forcato et al. (2017) Comparison of computational methods for Hi-C data analysis. <em>Nature methods</em> 14, 7, 679.</span> </li> <li id=ref-3 class="flex-c reference"> <span class=number>[3]</span> <span class=flex-g-1>Yardımcı and Noble. (2017) Software tools for visualizing Hi-C data. <em>Genome Biology</em>, 18, 1, 26.</span> </li> <li id=ref-4 class="flex-c reference"> <span class=number>[4]</span> <span class=flex-g-1>Durand et al. (2016) Juicebox provides a visualization system for Hi-C contact maps with unlimited zoom. <em>Cell Systems</em>, 3, 1, 99–101.</span> </li> <li id=ref-5 class="flex-c reference"> <span class=number>[5]</span> <span class=flex-g-1>Kerpedjiev et al. (2017) HiGlass: Web-based visual comparison and exploration of genome interaction maps. <em>bioRxiv</em>.</span> </li> </ol> </div> </div> <footer id=footer> <div class=content> <div class="flex-c flex-jc-c flex-a-c"> <div class="vcg-name flex-c flex-d-c"> <span>Visual</span> <span>Computing</span> <span>Group</span> </div> <svg class=logo-vcg> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#harvard-vcg></use> </svg> <svg class=logo-seas> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#harvard-seas></use> </svg> <svg class=logo-hms> <use xmlns:xlink=http://www.w3.org/1999/xlink xlink:href=images/icons.svg#harvard-hms></use> </svg> </div> <p class="centered m-t-1 copyright"> Copyright &copy; 2017 the President and Fellows of Harvard College. All content on this website is licensed under the <a href=http://creativecommons.org/licenses/by/4.0/legalcode target=_blank>Creative Commons Attribution license (CC BY)</a>. Copyright of the linked papers is with the publishers. </p> </div> <div id=more-at-lekschas-de class="flex-c flex-jc-c"> <a href=https://lekschas.de/research>More Projects by Fritz Lekschas</a> </div> </footer>  <script type="text/javascript" src="/index.js"></script></body> </html> 